example/src/screens/calendarScreen.tsx

```diff
@@ -173,6 +172,9 @@ const CalendarScreen = () => {
             [getDate(26)]: {color: 'gray'},
             [getDate(27)]: {endingDay: true, color: 'gray'}
           }}
+          disabledDaysIndexes={[3]}
+          disabledByWeekDays={[3]}
+          disableAllTouchEventsForDisabledDays
```

Comment:
A comment on this prop. onPress is still triggered when pressing a disabled date. Should we fix in this pr?

Reply:
It only affects markingType = 'period' so I prefer to address it in a different PR since there are many gaps btw the basic day and the period day that we should sync